src/widgets/table/table.rs

```diff
@@ -133,6 +133,23 @@ use crate::{
 /// Cell::from(Text::from("text"));
 /// ```
 ///
+/// Just as rows can be collected from iterators of `Cell`s, tables can be collected from iterators
+/// of `Row`s.  Note: While `Table::new` requires the column widths to be provided on construction,
+/// collection of an iterator into a `Table` does not have this safeguard.  If the column widths are
+/// not set with the `Table::widths`, the table will appear empty when drawn.
```

Comment:
I think this is no longer true, tables without width sould have columns of equal widths now.

Reply:
Ooh, nice!  I added this caveat since it was mentioned in #664.  Since #710 resolved the issue entirely, I'll remove this comment.

I'm wondering if #664 should be reverted entirely, now that the failure mode it guarded against is no longer present.  It would be a breaking change from 0.25 to 0.26, but would remove a breaking change from 0.24 to 0.26.

Reply:
I think we should keep it because the equal widths thing is just a failsafe. Most of the time you'll want to specify widths. Moreover, `default` already provide a behavior where widths are not set.

Reply:
I like having `new()` push users to provide the values for the most common use case for a type, so I want 664 to stay too.

Reply:
Good point, and that makes sense for guiding users to good practices.